microsoft-iis-mixin/README.md

```diff
@@ -0,0 +1,91 @@
+# Microsoft IIS Mixin
+
+Microsoft IIS mixin is a set of configurable Grafana dashboards and alerts.
+
+The Microsoft IIS mixin contains the following dashboards:
+
+- Microsoft IIS overview
+- Microsoft IIS applications
+
+and the following alerts:
+
+- MicrosoftIISHighNumberOfRejectedAsyncIORequests
+- MicrosoftIISHighNumberOf5xxRequestErrors
+- MicrosoftIISLowSuccessRateForWebsocketConnections
+- MicrosoftIISThreadpoolUtilizationNearingMax
+- MicrosoftIISHighNumberOfWorkerProcessFailures
+
+Default thresholds can be configured in `config,libsonnet`
+
+```js
+{
+  _config+:: {
+    alertsWarningHighRejectedAsyncIORequests: 20        // requests
+    alertsCriticalHigh5xxRequests: 5,                   // # of 500 level requests
+    alertsCriticalLowWebsocketConnectionSuccessRate: 80,// %
+    alertsCriticalHighThreadPoolUtilization: 90,        // %
+    alertsWarningHighWorkerProcessFailures: 10,         // failures
+  },
+}
+```
+
+## Microsoft IIS overview
+
+The Microsoft IIS overview dashboard provides details on traffic, connections, files sent/received, and access logs. [Promtail and Loki needs to be installed](https://grafana.com/docs/loki/latest/installation/) and provisioned for logs with your Grafana instance. Microsoft IIS access logs are found under `C:\inetpub\logs\LogFiles\W3SVC*\u_ex*.txt`. There will typically be a folder `W3SVC[siteID]` for each site. Under that folder there will be a number of log files corresponding to each date (Example: u_ex230313.txt).
```

Comment:
```suggestion
The Microsoft IIS overview dashboard provides details on traffic, connections, files sent/received, and access logs. [Promtail and Loki needs to be installed](https://grafana.com/docs/loki/latest/installation/) and provisioned for logs with your Grafana instance. Microsoft IIS access logs can be found via this glob `C:\inetpub\logs\LogFiles\W3SVC*\u_ex*.txt`. There will typically be a folder `W3SVC[siteID]` for each site. Under that folder there will be a number of log files corresponding to each date (Example: u_ex230313.txt).
```

For some reason reading that was hard to understand for me since its a file glob pattern not a folder. Feel free to reword I think theres probably a better wording to convey.